server/routes/authenticatedRoutes.ts

```diff
@@ -16,6 +17,7 @@ export default function routes(services: Services): Router {
   router.get(paths.RETURN_HOME, homeController.returnHome())
 
   router.use(addPrisonerRoutes(services))
+  router.use(addVisitorRoutes())
```

Comment:
Only thought was maybe putting the feature around this line to stop it even creating the routes?